ceph/ceph_admin/bootstrap.py

```diff
@@ -183,6 +191,25 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
         build_type = self.config.get("build_type")
         rhbuild = self.config.get("rhbuild")
 
+        # Support installation of the baseline cluster whose version is not available in
+        # CDN. This is primarily used for an upgrade scenario. This support is currently
+        # available only for RH network.
+        _rhcs_version = args.pop("rhcs-version")
+        _rhcs_release = args.pop("release")
+        if _rhcs_release and _rhcs_version:
+            _platform = "-".join(rhbuild.split("-")[1:])
+            _details = fetch_build_artifacts(_rhcs_release, _rhcs_version, _platform)
```

Comment:
would it be good, if we can make it as common under a definition in any module, so it could be called in module where build info needed.

Reply:
Thanks... yes, it would be helpful.

Can we pick this in another PR as the scope of this requirement was for baseline version?